### PR TITLE
[WIP] Add command line script for generating new keys

### LINKED ISSUE #338: Feature proposal: command line tool for generating a new key
In one of our projects that uses this library, we have a small PHP script `bin/generate-key.php` which simply generates a new key and prints it out:

```php
#!/usr/bin/env php
<?php

require __DIR__ . '/../vendor/autoload.php';

$key = Defuse\Crypto\Key::createNewRandomKey();
echo $key->saveToAsciiSafeString(), "\n";
```

I imagine others would have a similar need. Perhaps something like this could be added to this project as a [Composer vendor binary](https://getcomposer.org/doc/articles/vendor-binaries.md) so that one would simply need to run `vendor/bin/generate-defuse-key` (or whatever) to generate a new key.

Thoughts?

### PR DESCRIPTION
**Pull request by chriseskow**
Closes #338

### CODE REVIEW

**Comment by chriseskow** [comment]
Updated the tutorial with a reference to the script.

File: bin/generate-defuse-key [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+<?php
+
+use Defuse\Crypto\Key;
```
Comment (defuse) [inline bin/generate-defuse-key]: Doesn't the include need to happen before this statement will work?

Comment (paragonie-scott) [inline bin/generate-defuse-key]: No. Namespace imports go at the top.

**Comment by paragonie-scott** [comment]
Fixed a merge conflict introduced by pulling another branch.

**Comment by chriseskow** [comment]
@paragonie-scott Thanks! Anything else needed for this pull request?

File: bin/generate-defuse-key [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+foreach ([__DIR__ . '/../../../autoload.php', __DIR__ . '/../vendor/autoload.php'] as $file) {
+    if (file_exists($file)) {
+        require $file;
+        break;
+    }
+}
```
Comment (maidmaid) [inline bin/generate-defuse-key]:
```php
require file_exists(__DIR__.'/../vendor/autoload.php')
    ? __DIR__.'/../vendor/autoload.php'
    : __DIR__.'/../../../autoload.php'
;
```
More simple?

**Pull request merged**